### DISCUSSION TITLE
Add `main()` function to revset grammar

### DISCUSSION THREAD

**Discussion by pokey** [arxanas/git-branchless#579] [Ideas]
Would basically return `git config branchless.core.mainBranch`.  I think can be approximated for now by `heads(all() - draft())`?

**Comment by arxanas** [comment]
That sounds sensible. I created https://github.com/arxanas/git-branchless/pull/582 for this (and for the function `public()`, which is the opposite of `draft()` — the terminology comes from Mercurial).
